packages/iTwinUI-react/src/core/ComboBox/ComboBox.test.tsx

```diff
@@ -622,7 +622,45 @@ it('should use the latest onChange prop', async () => {
   expect(mockOnChange1).toHaveBeenNthCalledWith(1, 1);
 
   rerender(<ComboBox options={options} onChange={mockOnChange2} />);
+  await userEvent.tab({ shift: true }); // reopen menu
   await userEvent.click(screen.getByText('Item 2'));
   expect(mockOnChange2).toHaveBeenNthCalledWith(1, 2);
   expect(mockOnChange1).toHaveBeenCalledTimes(1);
 });
+
+it('should call onExpand and onCollapse when dropdown is opened and closed', () => {
+  const onExpand = jest.fn();
+  const onCollapse = jest.fn();
+  const { container } = renderComponent({
+    onShow: onExpand,
+    onHide: onCollapse,
+  });
+
+  const icon = container.querySelector('.iui-end-icon svg') as HTMLElement;
+  fireEvent.click(icon);
```

Comment:
I was trying to use `userEvent` for all the click actions

Reply:
Changed